### DISCUSSION TITLE
[Bug]: storybook won't start when tsconfig path aliases are used for `pandacss` generated `styled-system`

### DISCUSSION THREAD

**Discussion by bnn1** [storybookjs/storybook#26155] [Help] [answered]
### Describe the bug

Hey guys. I've setup storybook with path aliases support according to https://github.com/storybookjs/storybook/issues/21901#issuecomment-1518628256 and it works for all imports except for imports from `pandacss` generated  `styled-system`. I opened an issue in pandacss repo: https://github.com/chakra-ui/panda/issues/1157 but they said it's sb issue.

I'll just copy info from the original issue here:  

### The issue:
When I create a story from component that uses pandacss generated functions imported using absolute imports , I expect storybook to run normally but storybook won't start at all. When storybook is already running and I add a component that uses pandacss generated functions imported using absolute imports, I get "Module not found: Error: Can't resolve '@panda/css' in" error. 

### Steps to reproduce

1. `git clone git@github.com:bnn1/sb-panda-tsconfig.git`
2. run `pnpm install`
3. open `app/(components)/header/Header.tsx`
4. uncomment the first line and comment the second line
5. run `pnpm storybook`

### To Reproduce

https://github.com/bnn1/sb-panda-tsconfig

### System

```shell
System:
    OS: Linux 6.4 EndeavourOS
    CPU: (12) x64 AMD Ryzen 5 4600H with Radeon Graphics
  Binaries:
    Node: 18.14.2 - ~/.nvm/versions/node/v18.14.2/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 9.6.0 - ~/.nvm/versions/node/v18.14.2/bin/npm
```

**Comment by ioulian** [comment]
Also have the same problem

**Comment by ioulian** [comment, marked as answer]
@bnn1  Found it, this works:

```
webpackFinal: async (config) => {
  if (config.resolve) {
    config.resolve.alias = {
      ...config.resolve.alias,
      "@/styled-system": path.resolve(__dirname, "../styled-system"),
      "@": path.resolve(__dirname, "../src"),
    };
    config.resolve.plugins?.push(
      new TsconfigPathsPlugin(),
    );
  }

  return config;
},
```

Don't really know if TsconfigPathsPlugin is need or not
